docs/zoneddatetime.md

```diff
@@ -43,12 +43,14 @@ This range covers approximately half a million years. If `epochNanoseconds` is o
 
 Usage examples:
 
+<!-- prettier-ignore-start -->
 ```javascript
 // UNIX epoch in California
-new Temporal.ZonedDateTime(0n, Temporal.TimeZone.from('America/Los_Angeles), Temporal.Calendar.from('iso8601'))
+new Temporal.ZonedDateTime(0n, Temporal.TimeZone.from('America/Los_Angeles', Temporal.Calendar.from('iso8601')));
```

Comment:
```suggestion
new Temporal.ZonedDateTime(0n, Temporal.TimeZone.from('America/Los_Angeles'), Temporal.Calendar.from('iso8601'));
```

Reply:
Cool, good catch. Also, thanks for the spec text!  I'll fix this change and the other one and rebase and force-push.